driver/src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -81,6 +81,7 @@ public final class FuzzTargetRunner {
       if (notBootstrapLoader == null) {
         notBootstrapLoader = ClassLoader.getSystemClassLoader();
       }
+      notBootstrapLoader.setDefaultAssertionStatus(true);
```

Comment:
Ideally, we would want to enable assertions for anything transitively loaded by the fuzz target rather than the entire class loader. I don't know whether that's possible though. This is just porting over the native functionality almost 1-to-1, but when we develop the JUnit integration, we will have to revisit this for its side effects.

The existing tests already cover this, e.g. the various `JsonSanitizer*Fuzzer` examples rely on `assert`.